app/src/main/res/layout/settings_activity.xml

```diff
@@ -1,9 +1,130 @@
-<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    android:layout_width="match_parent"
-    android:layout_height="match_parent">
-
-    <FrameLayout
-        android:id="@+id/settings"
-        android:layout_width="match_parent"
-        android:layout_height="match_parent" />
-</LinearLayout>
```

Comment:
According to designdoc The design in MD built a simple interface, but I didn't completely follow the design in the file, because the interface was too crowded and it was difficult for users to control their hands to accurately locate each checkbox when using the mobile phone, so I only put half of the checkboxes on it.